_layouts/about.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'page-shell.html' %}
```

Comment:
Please move the layouts files to the templates folder like you'd normally have in a django app.